src/layout/header/MenuButton.tsx

```diff
@@ -1,19 +1,26 @@
-import { Button, ButtonProps } from '@mui/material';
+import { Box, Button, ButtonProps } from '@mui/material';
 import { Link, LinkProps } from 'react-router-dom';
 
 interface MenuButtonProps extends ButtonProps, Pick<LinkProps, 'to'> {
   isSelected: boolean;
 }
 
-export const MenuButton = ({ isSelected, ...rest }: MenuButtonProps) => (
-  <Button
+export const MenuButton = ({ isSelected, sx, ...rest }: MenuButtonProps) => (
+  <Box
     sx={{
-      whiteSpace: 'nowrap',
+      height: '100%',
+      display: 'flex',
       borderBottom: isSelected ? '0.375rem solid white' : 'none',
-      borderRadius: '0px',
-      mb: isSelected ? '-0.375rem' : '0rem', // Negates element re-size when border appears
-    }}
-    LinkComponent={Link}
-    {...rest}
-  />
+      boxShadow: '-1px 7px 4px -3px rgba(0,0,0,0.3)',
```

Comment:
For å sikra at bare den med fokus (undelinje) får shadow:
```suggestion
      boxShadow: isSelected ? '-1px 7px 4px -3px rgba(0,0,0,0.3)' : 'none',
```

Reply:
Ellers ser det bra ut, så tenker vi merger når det er fikset!